### PR TITLE
Small retroknowledge/primitive cleanup

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: kernel/typeops.ml [hunk 1]
```diff
@@ -664,17 +669,8 @@ let judge_of_case env ci pj cj lfj =
...
-      default_conv ~l2r:false CUMUL env mkSet t
-    with NotConvertible ->
-      CErrors.user_err Pp.(str"Was expecting the sort of this primitive type to be Set"))
-  | OT_op p ->
-    (try
-      default_conv ~l2r:false CUMUL env (type_of_prim env p) t
-    with NotConvertible ->
-      CErrors.user_err Pp.(str"Not the expected type for this primitive"))
+  let inft = type_of_prim_or_type env op_t in
+  (try default_conv ~l2r:false CUMUL env inft t
```
Comment (vbgl) [inline kernel/typeops.ml]: No need for parenthesis here.

**Review by vbgl — APPROVED** [review]

File: kernel/typeops.ml (hunk 1 above)
Comment (SkySkimmer) [reply kernel/typeops.ml]: Changed

**Review by ppedrot — APPROVED** [review]

**Comment by ejgallego** [comment]
18 days and no assignee, I may assign then if there is no action by other developers as per-policy.

**Comment by ejgallego** [comment]
But maybe @vbgl would like to assign this ?

**Comment by ejgallego** [comment]
Thanks @vbgl !

**Pull request merged**